### ISSUE TITLE
Demo application does not work.

### ISSUE THREAD

**Issue by remarkNS** [appreciated/vaadin-app-layout#4]
I did everything as explained, including bower install on step 2. After starting app, there are errors in console as shown in attached image.
![err1](https://user-images.githubusercontent.com/9980558/30607392-15657186-9d75-11e7-9e11-764ca2423562.png)

**Comment by appreciated** [comment]
Hello! I guess you have already tried to restart the application. The shown behaviour is to be expected when the bower components are not in their place. 
May I see the content of the following path `vaadin-app-layout\app-layout-addon\src\main\webapp\bower_components`? 
It should look like this: 
![bower_components](https://user-images.githubusercontent.com/1729845/30608416-8e35173a-9d78-11e7-9a05-c6983ccd0ffc.PNG)

**Comment by appreciated** [comment]
Sorry I guess I just found the error. The directory in the Readme was not correct.

**Comment by remarkNS** [comment]
Ok let me try everything again. I will let you know in 5 min.

**Comment by appreciated** [comment]
By the way why all the "hustle"? Because you can install the ready to use JAR via the Vaadin Directory (packaged _with_ all the dependencies).

**Comment by remarkNS** [comment]
Still does not work. 
![err2](https://user-images.githubusercontent.com/9980558/30610093-7a4469b0-9d7d-11e7-8878-66c7bc9efb4b.png)
I will try with jar from Vaadin Directory

**Comment by appreciated** [comment]
I have added the [Slack URL](https://join.slack.com/t/vaadin-app-layout/shared_invite/enQtMjQ0NDk0MTIyNjMwLTg3N2YxMzc3NDBhODY4ZGZhNDZlOTRmNzk3MjZhYzE3Nzg1YzQ3MGY0MDEyZTY2ZWRmMzlmOWM2MGE1OGRlZTM) to the wiki

**Comment by remarkNS** [comment]
It works now, thanks for the help. it looks and works very good.

**Comment by remarkNS** [comment]
Forgot to say, after I repeated all 3 steps again from the readme it started to work.

**Comment by appreciated** [comment]
Great! One more thing you won't be able to push anything if you followed the steps. If you want to create your own version I recommend to create your own fork and clone that instead that way you will have the possibility to use source code management. Have a great evening!

**Comment by rodrifmed** [comment]
Hi, 

I was using the app-layout fine with jetty and guice, but I needed to change the application to start to use vaadin spring-boot, and now the app-layout addon is not more working.

What is happening is similar to this picture that remarkNS posted. I saw in the sources in the chrome inspect and the folder bower_components is not there.

Should I change my pom to be similar to app-layout-demo? with the profile web-components?
